.gitattributes

```diff
@@ -60,6 +60,7 @@
 *.woff    -text diff
 *.woff2   -text diff
 
+
```

Comment:
@ChrispyAPAX Can you just run git checkout dev .gitattributes? So there isn't a diff for this file?

Reply:
done!